components/python/jinja2/Makefile

```diff
@@ -44,6 +45,9 @@ COMPONENT_TEST_DIR =	$(SOURCE_DIR)/tests
 COMPONENT_TEST_CMD =	$(PYTHON) -m pytest
 COMPONENT_TEST_ARGS =
 
+COMPONENT_TEST_TRANSFORMER = $(NAWK)
+COMPONENT_TEST_TRANSFORMS = "'/passed/ {gsub(\"in.* =*\",\"\"); print $0}'"
+
 # Manually added test dependencies
 REQUIRED_PACKAGES += library/python/flaky-37
 REQUIRED_PACKAGES += library/python/flaky-39
```

Comment:
While at it you should change the test dependencies, eg. for the first two:
TEST_REQUIRED_PACKAGES.python += library/python/flaky